maestro-studio/how-to-run-tests.md

```diff
@@ -0,0 +1,135 @@
+# How to run your first test
+
+To run tests with Maestro Studio, you just need to create a YAML file with the test flow. But first, you need to set up your environment for the platform you are developing for (Android, iOS, or Web).
+
+## Prerequisites
+
+- [Maestro Studio installed](/maestro-studio/how-to-install.md)
+
+**For Android tests**
+- [Android Studio](https://developer.android.com/studio/install).
+- [Android Virtual Device (emulator) configured](https://developer.android.com/studio/run/emulator#avd).
+
+**For iOS tests**
+- [XCode installed](https://apps.apple.com/us/app/xcode/id497799835?mt=12).
+- [XCode command line tools installed](https://developer.apple.com/xcode/resources/).
+
+{% hint style="success" %}
+The XCode already provides a simulator, if you need to install a differente one than the default, follow the instructions on the Apple Official documentation: [Adding additional simulators](https://developer.apple.com/documentation/safari-developer-tools/adding-additional-simulators).
+{% endhint %}
+
+## Creating your first YAML file
+
+Once you have installed and set up your development environment, you can proceed to Maestro Studio and start creating your first YAML file.
+
+1. Open Maestro Studio.
+2. Click the **No device connected** button to open the panel.
+
+![Open the device selection panel](/images/maestro-studio/no-device.png)
+
+3. On the panel, selecte the device you want to run your tests.
+
+![Select the device to run the test](/images/maestro-studio/select-device.png)
+
+4. On the Maestro Studio, click **New Test** option to open the **Add a new test to your workspace** window.
+
+![Open the Add new test to your workspace window](/images/maestro-studio/add-test.png)
+
+5. On the **Add a new test to your workspace** window, select the type of test (Mobile Test, Web Test, or JavaScript File), and fill in:
+    - **Name**: Name for your YAML file.
+    - **App Id**: From the dropdown menu, select the App Id for testing.
+        - You can also click the **Scan file for App Id** option. This allows you to indicate an app file to test (such as an `.apk` for Android or a `.zip` for iOS).
```

Comment:
I can't remember if this functionality accepts `.zip` or `.app/` dir or both for iOS?